core/src/test/java/org/bitcoinj/core/SegwitAddressTest.java

```diff
@@ -68,7 +69,7 @@ public void example_p2wpkh_mainnet() {
     public void example_p2wsh_mainnet() {
         String bech32 = "bc1qrp33g0q5c5txsp9arysrx4k6zdkfs4nce4xj0gdcccefvpysxf3qccfmv3";
 
-        SegwitAddress address = SegwitAddress.fromBech32(BitcoinNetwork.MAINNET, bech32);
```

Comment:
Why is this being changed?

(multiple occurrences)

Reply:
This probably happened because of a rebase (and me not noticing during the rebase)

Reply:
Fixed in 04a578a7670dcfb9b209477d935bb205114d491b